maven-plugin/src/main/java/org/openmrs/maven/plugins/model/Server.java

```diff
@@ -658,11 +660,17 @@ public void setIncludeDemoData(boolean includeDemoData) {
         setParam(PROPERTY_DEMO_DATA, String.valueOf(includeDemoData));
     }
 
-    public String getPlatformVersion(){
+    @Deprecated
+    public String getOpenmrsCoreVersion(){
         return getParam(PROPERTY_PLATFORM);
     }
-    public void setPlatformVersion(String platformVersion){
-        setParam(PROPERTY_PLATFORM, platformVersion);
+
+    public void saveOpenmrsCoreVersion(String version) throws MojoExecutionException {
```

Comment:
Saving changes in distro properties file, but not changing openmrs-server properties file seems a bit incosistent and can possibly introduce bugs